lib/mongo/error/invalid_max_connecting.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Copyright (C) 2014-present MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  class Error
+    # Exception that is raised when trying to create a client with an invalid
+    #   max_connecting option.
+    #
+    # @since 2.4.2
```

Comment:
Should this be 2.19.0 (not 2.4.2)?

Reply:
This is a copy/paste artifact, too. We do not use @since tag at all now. Removed, thank you.